### PR TITLE
Filter registered sessions on event detail page

### LINKED ISSUE #482: Filter registered sessions on event detail page
## Description

Filter registered sessions on the event details page. Registered sessions under an event are those where a character belonging to the logged in user is included in the list of characters registered to a session.

### PR DESCRIPTION
**Pull request by bcobo341**
## Description
Closes #482 
- Filter registered sessions

### CODE REVIEW

File: resources/js/Pages/Event/Detail/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export {default as CharacterDetail} from './EventDetail'
```
Comment (pascwhale) [inline resources/js/Pages/Event/Detail/index.ts:1]:
```suggestion
export {default as EventDetail} from './EventDetail'
```

**Review by pascwhale — CHANGES_REQUESTED** [review]
Everything looks good except for on change I added as a suggestion.

**Review by pascwhale — APPROVED** [review]
Thank you for addressing the requested change. The frontend files look correct.

**Review by m-triassi — APPROVED** [review]
Super clean Backend implementation, nice work!

**Pull request merged**